docs/contribute.md

```diff
@@ -54,8 +54,9 @@ cd ./src
 ### Editors
 
 [Using Visual Studio Code](https://code.visualstudio.com/)
-[Using Visual Studio](https://code.visualstudio.com/docs)  
 
+[Using Visual Studio](https://code.visualstudio.com/docs)  
+We recommend using [Visual Studio 2019](https://visualstudio.microsoft.com/vs/) since it has built in support for .NET and other libraries that are helpful for contributing and running this repo while newer versions of Visual Studio does not have support for these libraries as of yet. 
```

Comment:
I would expect the older VS version might not work with dotnet core, not the newer version.

What error do you get?

I would not say `We recommend using [Visual Studio 2019]` since VsCode is a better IDE for this repo since the runner is for Windows/Linux/macOS, but you can only use VS in Windows.

Reply:
I had a lot of issues with omnisharp like: https://github.com/OmniSharp/omnisharp-vscode/issues/2507. But when I reinstall Visual Studio Code, the error disappeared and I wasn't able to replicate it so I'm not sure what's going on. I'll close this PR for now :P